packages/components/src/radio-control/index.tsx

```diff
@@ -75,24 +79,24 @@ export function RadioControl(
 	const onChangeValue = ( event: ChangeEvent< HTMLInputElement > ) =>
 		onChange( event.target.value );
 
-	// Use `useBaseControlProps` to get the id of the help text.
-	const {
-		controlProps: { 'aria-describedby': helpTextId },
-	} = useBaseControlProps( { id, help } );
-
 	if ( ! options?.length ) {
 		return null;
 	}
 
 	return (
-		<BaseControl
-			__nextHasNoMarginBottom
-			label={ label }
+		<fieldset
 			id={ id }
-			hideLabelFromVision={ hideLabelFromVision }
-			help={ help }
-			className={ clsx( className, 'components-radio-control' ) }
+			className={ clsx( 'components-radio-control', className ) }
```

Comment:
Reordering the classnames now can lead to 3rd party code specificity issues where selectors aren't specific enough. Should we do it separately and do some research/testing?

Reply:
I agree with you, mostly. However, my recommendation was exactly that to bringe the gap between "hope" and "confirm" we'll need some testing, and that can also happen in another PR 😉

Reply:
Reverted in https://github.com/WordPress/gutenberg/pull/64582/commits/86917565a08815deeee80d3e009a501dc7aadf99

Reply:
FWIW, seconding that changing class name order like this will not affect anything, regardless of how the consumer CSS is written or loaded.